server/internal/infrastructure/db/postgres/migration/20250526124117_init.up.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE IF NOT EXISTS round (
+    id VARCHAR PRIMARY KEY,
+    starting_timestamp BIGINT NOT NULL,
+    ending_timestamp BIGINT NOT NULL,
+    ended BOOLEAN NOT NULL,
+    failed BOOLEAN NOT NULL,
+    stage_code INTEGER NOT NULL,
+    txid VARCHAR NOT NULL,
+    connector_address VARCHAR NOT NULL,
+    version INTEGER NOT NULL,
+    swept BOOLEAN NOT NULL,
+    vtxo_tree_expiration BIGINT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS tx_request (
+    id VARCHAR PRIMARY KEY,
+    round_id VARCHAR NOT NULL,
+    FOREIGN KEY (round_id) REFERENCES round(id)
+);
+
+CREATE TABLE IF NOT EXISTS receiver (
+    request_id VARCHAR NOT NULL,
+    pubkey VARCHAR,
+    onchain_address VARCHAR,
+    amount BIGINT NOT NULL,
+    FOREIGN KEY (request_id) REFERENCES tx_request(id),
+    PRIMARY KEY (request_id, pubkey, onchain_address)
+);
+
+CREATE TABLE IF NOT EXISTS tx (
+    txid VARCHAR PRIMARY KEY,
+    tx TEXT NOT NULL,
+    round_id VARCHAR NOT NULL,
+    type VARCHAR NOT NULL,
+    position INTEGER NOT NULL,
+    tree_level INTEGER,
+    parent_txid VARCHAR,
+    is_leaf BOOLEAN,
+    FOREIGN KEY (round_id) REFERENCES round(id)
+);
+
+CREATE TABLE IF NOT EXISTS vtxo (
+    txid VARCHAR NOT NULL,
+    vout INTEGER NOT NULL,
+    pubkey VARCHAR NOT NULL,
+    amount BIGINT NOT NULL,
+    round_tx VARCHAR NOT NULL,
+    spent_by VARCHAR NOT NULL,
+    spent BOOLEAN NOT NULL,
+    redeemed BOOLEAN NOT NULL,
+    swept BOOLEAN NOT NULL,
+    expire_at BIGINT NOT NULL,
+    created_at BIGINT NOT NULL,
+    request_id VARCHAR,
+    redeem_tx TEXT,
```

Comment:
this can be dropped, the virtual tx is stored in the dedicated table now

Reply:
redeem_tx from vtxo domain is used in codebase, shall we postpone this change for another PR?

Reply:
i'm saying to drop it only from the vtxo table as it would be basically duplicated.